tests/test_ingress.py

```diff
@@ -4,12 +4,44 @@
 from . import supported_k8s_versions
 
 
+def ingress_assertions_v1(doc):
+    assert doc["apiVersion"] == "networking.k8s.io/v1"
+    assert "RELEASE-NAME-astro-ui" in [
+        name[0]
+        for name in jmespath.search(
+            "spec.rules[*].http.paths[*].backend.service.name", doc
+        )
+    ]
+    assert "astro-ui-http" in [
+        port[0]
+        for port in jmespath.search(
+            "spec.rules[*].http.paths[*].backend.service.port.name", doc
+        )
+    ]
+
+
+def ingress_assertions_v1beta1(doc):
```

Comment:
I'm new to the test codebase so not sure if this is following a pattern, but this would read better to me if it was something more like `assert_ingress_v1beta1` (verb vs. noun).

Reply:
should be good "test_ingress_assertions_v1"

Reply:
These are helper functions, not individual test cases. We're not only asserting that it's v1beta1, but also that the data structure is the needed structure for the given version.